solution/ui/e2e/cypress/e2e/subjects.spec.cy.js

```diff
@@ -830,14 +774,6 @@ describe("Find by Subjects", () => {
         cy.url().should("include", "/statutes");
     });
 
-    it("should have the correct labels for public and internal documents", () => {
-        cy.checkPolicyDocs({
```

Comment:
This is tested thoroughly in the search page test suite, no need to duplicate here